### ISSUE TITLE
Upgrade pyspark version in evidence data source parsers

### ISSUE THREAD

**Issue by ireneisdoomed** [opentargets/issues#3334]
We want to upgrade the PySpark version in the [evidence source parsers](https://github.com/opentargets/evidence_datasource_parsers) to 3.5 (currently on 3.3.1)

## Tasks

- [ ] Run evidence generation for 24.09 with the latest PySpark release
- [ ] We have written custom functions like [unpivot](https://github.com/opentargets/evidence_datasource_parsers/pull/192#discussion_r1626348910) that have been recently added, so it'd be good to move them to the Spark API.

**Comment by prashantuniyal02** [comment]
@opentargets/data-team , do you think if this is still relevant. I can see that the version was bumped from 3.3.1 to 3.3.2

**Comment by ireneisdoomed** [comment]
Yes, will be done with the integration of the parsers as part of the orchestration (not for 25.06)

**Comment by d0choa** [comment]
@opentargets/data-team I'm closing this ticket. I assume that you will update all dependencies in the evidence source parsers as soon as you make any changes there. Feel free to reopen if necessary
